projects/plugins/jetpack/extensions/blocks/ai-assistant/upgrade-prompt.js

```diff
@@ -0,0 +1,25 @@
+/*
+ * External dependencies
+ */
+import { __ } from '@wordpress/i18n';
+import React from 'react';
+/*
+ * Internal dependencies
+ */
+import { Nudge } from '../../shared/components/upgrade-nudge';
+
+const UpgradePrompt = () => {
+	return (
```

Comment:
We should return `null` when it isn't a Jetpack site (at least for now)

```suggestion
const UpgradePrompt = () => {
	if ( isAtomicSite() || isSimpleSite() ) {
		return null;
	}
	
	return (
```

Reply:
You need to requite the functions in the `External dependencies` block, too.

```es6
import { isAtomicSite, isSimpleSite } from '@automattic/jetpack-shared-extension-utils';
```